ChangeLog.md

```diff
@@ -1,5 +1,266 @@
 # Changelog for haskell-language-server
 
+## 1.8.0.0
+
+- Binaries for GHC 9.2.3 and GHC 9.2.4
+- Initial support for GHC 9.4 with binaries for GHC 9.4.1 and GHC 9.4.2
+- Startup time and performance improvements on projects using Template Haskell by serializing intermediate core (#2813)
+- Improvements for hls-class-plugin (#2920, @July541) 
+- The new hls-gadt-plugin (#2899, @July541)
+- Moving code actions from ghcide to the new hls-refactor-plugin (#3091, @wz1000)
+- Many more improvements and bug fixes thanks to our contributors!
```

Comment:
Worth highlighting the memory savings achieved by @kokobd migrating  file paths to a compact representation